digital-koiyu/usr/share/asteroid-launcher/watchfaces/digital-koiyu.qml

```diff
@@ -0,0 +1,435 @@
+/*
+ * Copyright (C) 2023 - Arseniy Movshev <dodoradio@outlook.com>
+ * All rights reserved.
+ *
+ * You may use this file under the terms of BSD license as follows:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the author nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import QtQuick 2.1
+import QtGraphicalEffects 1.0
+import Nemo.Mce 1.0
+
+Item {
+    id: root
+    property string imgPath: "../watchfaces-img/digital-koiyu/"
+    property int hours: use12H.value ? wallClock.time.getHours() % 12 : wallClock.time.getHours()
+    property string minutes: wallClock.time.getMinutes() < 10 ? "0" + wallClock.time.getMinutes() : wallClock.time.getMinutes()
+    property string mainDisplayValue: (hours < 10 ? " " : "") + hours + minutes + " "//this if statement gets rid of the leading 0 on the hours
```

Comment:
Which if statement ? I suppose you mean that ternary condition which pads the hour with a space ?